### PR TITLE
Organiz

### PR DESCRIPTION
**Pull request by Neustradamus**
- Sort include
- Remove: Duplicates
- Remove: Some empty lines
- Add: Psi Team copyrights

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -1905,9 +1905,9 @@ public:
     	return false;
 #else
     	QString str =
+    		"#include<X11/extensions/scrnsaver.h>\\n"
```
Comment (Ri0n) [inline configure]: This is a generated file. It doesn't make sense to change it manually.

File: src/AutoUpdater/AutoUpdater.h [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (C) 2008 Remko Troncon
+ * Copyright (C) 2001-2019  Psi Team
```
Comment (Ri0n) [inline src/AutoUpdater/AutoUpdater.h]: This file got real code modifications in 2015. 
2019 is not true. I guess the same for many other files.

File: src/accountregdlg.h [hunk 3]
```diff
@@ -1,6 +1,7 @@
 /*
  * accountregdlg.h
- * Copyright (C) 2001, 2002, 2006  Justin Karneges, Remko Troncon
+ * Copyright (C) 2001-2019  Psi Team
```
Comment (Ri0n) [inline src/accountregdlg.h]: I personally don't like this bloated copyright stuff. Look at the line below, it's reflects the history good enough.
And yes some real code logic changes were made in 2017, not 2019.

File: src/activeprofiles_win.cpp [hunk 4]
```diff
@@ -17,15 +18,15 @@
...
  */
 
-#include "activeprofiles.h"
-#include "applicationinfo.h"
-#include "psicon.h"
-
 #include <QCoreApplication>
-#include <QWidget>
 #include <QTimer>
+#include <QWidget>
```
Comment (Ri0n) [inline src/activeprofiles_win.cpp]: Usually it's a good practice to include 3rdparty headers after application headers. This validates the headers for completeness. You break this rule here.
And be especially careful with `windows.h`. Usually it's order of inclusion may matter a lot. I hope you checked dthe code compiles well.

File: src/statuspreset.cpp [hunk 5]
```diff
@@ -17,10 +18,11 @@
  *
  */
 
+#include "statuspreset.h"
```
Comment (Ri0n) [inline src/statuspreset.cpp]: this one weird

File: src/sxe/sxeedit.cpp [hunk 6]
```diff
@@ -18,6 +19,7 @@
  */
 
 #include "sxeedit.h"
+
```
Comment (Ri0n) [inline src/sxe/sxeedit.cpp]: empty line? for what?

**Comment by Ri0n** [comment]
I'm rejecting it since most of this is invalid and I'd prefer to not combine all kinds of changes in one huge PR

File: configure (hunk 1 above)
Comment (Neustradamus) [reply configure]: If this point of view, we can remove the file ;)

File: src/AutoUpdater/AutoUpdater.h (hunk 2 above)
Comment (Neustradamus) [reply src/AutoUpdater/AutoUpdater.h]: It is not a problem.
And I have edited this file in 2019 for this perfect change :)

File: src/accountregdlg.h (hunk 3 above)
Comment (Neustradamus) [reply src/accountregdlg.h]: It is not a problem.
And I have edited this file in 2019 for this perfect change :)

File: src/activeprofiles_win.cpp (hunk 4 above)
Comment (Neustradamus) [reply src/activeprofiles_win.cpp]: Same logic is take into account for all files.

File: src/statuspreset.cpp (hunk 5 above)
Comment (Neustradamus) [reply src/statuspreset.cpp]: It is already in place to several files.
The .h #include of the .cpp file before <include>.

File: src/sxe/sxeedit.cpp (hunk 6 above)
Comment (Neustradamus) [reply src/sxe/sxeedit.cpp]: Yes to use the same logic in all places, already in place to several files.
And to seperate the .h #include of the same file, there is no <include> in this file.

**Comment by Neustradamus** [comment]
Like requested by @Ri0n, I have removed the copyright.
  * Copyright (C) 2001-2019  Psi Team

**Comment by Vitozz** [comment]
Changes from 01-08-2019 did not pass any of the Travis-CI tests. This means that on real build machines, the build will fail. Therefore, this pull  request requires correction of compilation errors. On my opinion sorting of  #include and namespaces according to the alphabet without proper tests it is not a good idea.

**Comment by Neustradamus** [comment]
@Vitozz: The last has been done yesterday...

**Comment by Vitozz** [comment]
@Neustradamus you have to resolve 3 conflicts after master changes
systeminfo.cpp - you have to leave master version
textutil.cpp - you have to leave master version instead of yours
busywidget.cpp - you can move master part upper but do not reorder #ifdef and #include from master (this order is very important)

**Pull request merged**